Components/Diary.js

```diff
@@ -1,27 +1,39 @@
-import React from 'react';
-import { Text, View, StyleSheet } from 'react-native';
-import moment from "moment"
+import React from "react";
+import { Text, View, ScrollView } from "react-native";
+import moment from "moment";
+import styles from "./Styling/diaryStyling";
+import HeaderBar from "./HeaderBar";
 
 const DiaryScreen = props => (
-    <View style={styles.diary}>
-          {props.screenProps.quiz.map(({completed_at, mood, stiffness, slowness, tremor, id}) => (
-            <View key={id}>
-              <Text>Taken: {moment(completed_at).format('DD-MM-YYYY HH:MM')}</Text>
-              <Text>Mood score: {mood}</Text>
-              <Text>Stiffness score: {stiffness}</Text>
-              <Text>Slowness score: {slowness}</Text>
-              <Text>Tremor score: {tremor}</Text>
-            </View>
-          ))}
-        </View>
+  <View style={styles.diaryMain}>
+    <HeaderBar />
+    <Text style={styles.diaryTitle}>Quiz Results</Text>
+    <ScrollView style={styles.diaryScrollView}>
+      {props.screenProps.details.quiz
+        ? props.screenProps.details.quiz.map(
+            ({ completed_at, mood, stiffness, slowness, tremor, id }) => {
+              return (
+                <View key={id} style={styles.diarySingle}>
```

Comment:
I'd be tempted to deconstruct all these stylesheets somewhere else to drop the amount of "styles." but this works and is fine.